bodhi/tests/server/base.py

```diff
@@ -136,10 +137,11 @@ class BaseTestCase(unittest.TestCase):
         'cors_connect_src': 'http://0.0.0.0:6543',
         'cors_origins_ro': 'http://0.0.0.0:6543',
         'cors_origins_rw': 'http://0.0.0.0:6543',
-        'sqlalchemy.url': DEFAULT_DB
+        'sqlalchemy.url': DEFAULT_DB,
+        'f17.autoupdates.from_tags': 'f17-updates-candidate',
```

Comment:
I think this was used in the first implementation of this PR but should not be required anymore :smiley:

Reply:
Oops. Will remove.

Reply:
Removed.